### ISSUE TITLE
JKube E2E Tests (CRC-OKD) pipeline is not working

### ISSUE THREAD

**Issue by manusa** [eclipse-jkube/jkube-integration-tests#392]
Not sure if this is working:
https://github.com/eclipse-jkube/jkube-integration-tests/actions/runs/11291874838

_Originally posted by @manusa in https://github.com/eclipse-jkube/jkube-integration-tests/issues/387#issuecomment-2407299798_

**Comment by rohanKanojia** [comment]
Looks like something has changed on GitHub actions again, I see these failures now:
```
E: Unable to locate package virtiofsd
Error: Process completed with exit code 100.
```

**Comment by manusa** [comment]
It's not that it has changed, it's the operating system that's different for each execution

https://github.com/eclipse-jkube/jkube-integration-tests/actions/runs/11432967006/job/31804528762#step:1:4
```
Operating System
  Ubuntu
  22.04.5
  LTS
```

The pipeline needs to ensure that the dependency is installed only if required

**Comment by manusa** [comment]
Closed by #395
